ui/src/components/common/LandingPage.tsx

```diff
@@ -25,6 +25,7 @@ const LandingPage: React.FC<Props> = ({
     <Page
         sideNav={sideNav}
         onLogout={onLogout}
+        isLandingPage={true}
```

Comment:
This can just be `isLandingPage`, since the existence of the prop will count as true